README.md

```diff
@@ -27,14 +28,16 @@ This library depends on
 - [``matplotlib``](https://matplotlib.org/stable/)
 - [``urllib3``](https://urllib3.readthedocs.io/en/stable/)
 - [``urchin``](https://github.com/fishbotics/urchin)
+- [``jaxsim``](https://github.com/ami-iit/jaxsim)
 
 To install you can use the following commands
 
 
 ```
-conda create -n comododev -c conda-forge adam-robotics idyntree bipedal-locomotion-framework=0.19.0 mujoco mujoco-python numpy mujoco-python-viewer matplotlib urllib3 urchin resolve-robotics-uri-py notebook
+conda create -n comododev -c conda-forge adam-robotics idyntree bipedal-locomotion-framework=0.19.0 mujoco mujoco-python numpy mujoco-python-viewer matplotlib urllib3 urchin resolve-robotics-uri-py notebook jaxsim
```

Comment:
The requirements list can probably be simplified as for example, `numpy` is already a dependency of many of the requested packages (e.g. `adam-robotics`), so it can be excluded, `idyntree` is a dependency of `blf` and so on
```suggestion
conda create -n comododev -c conda-forge adam-robotics bipedal-locomotion-framework=0.19.0 mujoco-python-viewer matplotlib urllib3 urchin notebook jaxsim
```